test/groovy/MTABuildTest.groovy

```diff
@@ -30,95 +30,52 @@ public class MTABuildTest extends BasePipelineTest {
-        otherDir = tmp.newFolder().toURI().getPath()[0..-2] //omit final '/'
 
-        helper.registerAllowedMethod('readYaml', [Map], {
-            m ->
-                return new Yaml().load((m.file as File).text)
-        })
-        helper.registerAllowedMethod("dir", [String, Closure], {
-            s, c ->
-                currentDir = "${currentDir}/${s}"
-                c()
-        })
         helper.registerAllowedMethod('pwd', [], { currentDir } )
-        helper.registerAllowedMethod("withEnv", [List.class, Closure.class],
-                { l, c ->
-                    mtaBuildShEnv = l
-                    c()
-                })
         helper.registerAllowedMethod('error', [String], { s -> throw new hudson.AbortException(s) })
+        helper.registerAllowedMethod('fileExists', [String], { String path ->
+            return path.endsWith('.pipeline/config.yml')
+        })
 
         binding.setVariable('PATH', '/usr/bin')
-        binding.setVariable('JAVA_HOME', '/opt/java')
-        binding.setVariable('env', [:])
 
-        mtaBuildScript = loadScript("mtaBuild.groovy").mtaBuild
-        cpe = loadScript('commonPipelineEnvironment.groovy').commonPipelineEnvironment
+        mtaBuildScript = loadScript('mtaBuild.groovy').mtaBuild
     }
 
 
     @Test
-    public void straightForwardTest(){
-
-        binding.getVariable('env')['MTA_JAR_LOCATION'] = '/opt/mta'
+    public void environmentPathTest(){
 
         new File("${currentDir}/mta.yaml") << defaultMtaYaml()
 
-        def mtarFilePath = mtaBuildScript.call(script: [commonPipelineEnvironment: cpe],
-                                         buildTarget: 'NEO')
-
-        assert jscr.shell[0] =~ /sed -ie "s\/\\\$\{timestamp\}\/`date \+%Y%m%d%H%M%S`\/g" ".*\/mta.yaml"$/
-
-        assert jscr.shell[1].contains("PATH=./node_modules/.bin:/usr/bin")
+        mtaBuildScript.call(buildTarget: 'NEO')
 
-        assert jscr.shell[1].contains(' -jar /opt/mta/mta.jar --mtar ')
-
-        assert mtarFilePath == "${currentDir}/com.mycompany.northwind.mtar"
-
-        assert jlr.log.contains( "[mtaBuild] MTA JAR \"/opt/mta/mta.jar\" retrieved from environment.")
+        assert jscr.shell[1].contains('PATH=./node_modules/.bin:/usr/bin')
     }
 
 
     @Test
     public void mtarFilePathFromCommonPipelineEnviromentTest(){
 
-        binding.getVariable('env')['MTA_JAR_LOCATION'] = '/opt/mta'
-
         new File("${currentDir}/mta.yaml") << defaultMtaYaml()
 
-        mtaBuildScript.call(script: [commonPipelineEnvironment: cpe],
-                      buildTarget: 'NEO')
-
-        def mtarFilePath = cpe.getMtarFilePath()
+        def mtarFilePath = mtaBuildScript.call(buildTarget: 'NEO')
 
         assert jscr.shell[0] =~ /sed -ie "s\/\\\$\{timestamp\}\/`date \+%Y%m%d%H%M%S`\/g" ".*\/mta.yaml"$/
-
-        assert jscr.shell[1].contains("PATH=./node_modules/.bin:/usr/bin")
-
-        assert jscr.shell[1].contains(' -jar /opt/mta/mta.jar --mtar ')
```

Comment:
I guess the asserts has been removed since they do not fit into the new test name.
Are these asserts (which are IMO valid) for the straight forward use-case checked somewhere else?

The most problematic point here is: this does not relate to the focus of the PR. I would like to suggest to provide two pull requests.

1.) PR changing the tests and keeping the configuration framework.
2.) PR changing the configuration framework and keeping the tests.